### PR TITLE
feat: added card component

### PR DESCRIPTION
**Pull request by adidoshi**
Hey! I created card component, please review it.

### CODE REVIEW

**Review by srishti-maurya — APPROVED** [review]
looks good

File: components/card/Card.html [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+              style="
+                width: 100%;
+                height: 160px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
```
Comment (tanveertkd) [inline components/card/Card.html:152]: You can move your iframe styling to one of the common stylesheet's. It'll help with maintenance and keep and html clean.

**Review by techieshubham30 — APPROVED** [review]

**Review by tanveertkd — APPROVED** [review]
Just the one suggestion apart from that the whole thing looks really good!

File: components/card/Card.html (hunk 1 above)
Comment (adidoshi) [reply components/card/Card.html:152]: Ya that would make the code look more clean, thanks.

**Comment by nshanbhag20001412** [comment]
Iframes are not responsive horizontally. Thought it's not necessary to make the app responsive now, still thought to let you know. The rest looks great!

**Review by nshanbhag20001412 — APPROVED** [review]

**Pull request merged**